File: ex2/js/main.js
console.log("conected");

let fibSeries = [1,2]
let maxNumber = 4000000
let x = 0
answerSeries = []

while (x <= maxNumber) {
  x = (fibSeries[(fibSeries.length - 1)]) + (fibSeries[(fibSeries.length - 2)])
  if (x < maxNumber) {
    fibSeries.push(x)
  }
}

answerSeries = fibSeries.filter( function(x) {
  return x % 2 === 0
})

answer = answerSeries.reduce( function (a,b) {
  return (a + b)
}, 0 ) // which is the default value

console.log(answer);

File: ex2/main.py
print("hello python")

fib_series = [1,2]
x = 0
max_number = 4000000
answer = 0

while (x <= max_number):
    x = (fib_series[( len(fib_series) -1 )]) + (fib_series[( len(fib_series) -2 )])
    if (x < max_number): fib_series.append(x)


for x in fib_series:
    if x % 2 == 0: answer = answer + x

print(answer)
